source/components/recipe-details/detailpage.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Recipe Detail Page</title>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <link rel="stylesheet" type="text/css" href="http://fonts.googleapis.com/css?family=Ubuntu:regular,bold&subset=Latin">
+  </head>
+  <body>
+    <header>
+      
+    </header>
+    <main>
+      <div class="recipe">
+        <!-- recipe image-->
+        <img 
+          class="recipe-image"
+          src="../../static/home-page/demorecipe.jpg"
+        />
+        <!-- the box on the right side of image-->
+        <div class="title-author-box"></div>
+        <!-- recipe name, author, date and website-->
+        <div>
+          <h1 class="recipe-name">Vegetable Dish</h1>
+          <h3 class="author-date">
+              by Anonymous
+              <br>
+              9-11-2021  
+              <br> 
+              https://amazingrecipe.com
+          </h3>
+        </div>
+        <!-- Description box-->
+        <div class="description-box">
+          <div class="description-title">
+            Description:
+          </div>
+          <div class="description">
+            Description here asdfadfasdfasdfasdfasdfasdfasdfasdfasdfdafadfasfasdfasdfasdfasfafasdfasdfafdasdfafsdfasdf
+            asdjhfkajshdfjahsdfkjhaskdjhfkajsdhfkajhsdfkjsahfkajshdfjkhaskdjhfakjshdfkajshdfkajhdasdsdfasf
+            sdkjahfkjashdfkjahsdkjfhajksdhfkajshdfk
+          </div>
+        </div>
+        <!-- ingredients box-->
+        <div class="ingredients">
+          <div class="ing-title">
```

Comment:
Please don't abbreviate class names. See below:
```suggestion
          <div class="ingredients-title">
```